tests/integration_tests/lightgbm_tuner_tests/test_alias.py

```diff
@@ -37,40 +41,56 @@ def test_handling_alias_parameter() -> None:
     assert params["min_data_in_leaf"] == 0.2
 
 
-def test_handling_alias_metrics() -> None:
-
-    for alias in [
-        "lambdarank",
-        "rank_xendcg",
-        "xendcg",
-        "xe_ndcg",
-        "xe_ndcg_mart",
-        "xendcg_mart",
-        "ndcg",
-    ]:
-        lgbm_params = {"metric": alias}
-        _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "ndcg"
-
-    for alias in ["mean_average_precision", "map"]:
-        lgbm_params = {"metric": alias}
-        _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "map"
-
-    for alias in ["regression", "regression_l2", "l2", "mean_squared_error", "mse"]:
-        lgbm_params = {"metric": alias}
-        _handling_alias_metrics(lgbm_params)
-        assert lgbm_params["metric"] == "l2"
-
-    lgbm_params = {}
-    _handling_alias_metrics(lgbm_params)
-    assert lgbm_params == {}
-
-    lgbm_params = {"metric": "auc"}
-    _handling_alias_metrics(lgbm_params)
-    assert lgbm_params["metric"] == "auc"
-
-    for alias in ["l2_root", "root_mean_squared_error", "rmse"]:
-        lgbm_params = {"metric": alias}
+@pytest.mark.parametrize(
+    "aliases, expect",
+    [
+        (
+            [
+                "ndcg",
+                "lambdarank",
+                "rank_xendcg",
+                "xendcg",
+                "xe_ndcg",
+                "xe_ndcg_mart",
+                "xendcg_mart",
+            ],
+            "ndcg",
```

Comment:
Thank you for adding the expected metric name to the aliases. Now we can check the identical transformation :+1: